.github/workflows/bindings-generator.yml

```diff
@@ -0,0 +1,137 @@
+name: Auto-generate Bindings
+
+on:
+  push:
+    branches:
+      - 'generate/aws-lc-sys-*'
+
+env:
+  GIT_AUTHOR_NAME: "aws-lc-sys-bindings-generator"
+  GIT_AUTHOR_EMAIL: "aws-lc-github-bot@amazon.com"
+  GIT_COMMITTER_NAME: "aws-lc-sys-bindings-generator"
+  GIT_COMMITTER_EMAIL: "aws-lc-github-bot@amazon.com"
+
+jobs:
+  collect-symbols-and-commit:
+    if: github.repository == 'aws/aws-lc-rs'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ ubuntu-latest, macos-12, macos-13-xlarge ]
```

Comment:
why is macos in here twice?

Reply:
`macos-12` is x86-64, `macos-13-xlarge` is aarch64.

Reply:
oh for real? that's not obvious at all 😄

Reply:
Yeah, it's a mess: https://github.blog/2023-10-02-introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/#how-to-use-the-runner
> To try the new Apple silicon macOS larger runner, update the `runs-on:` key in your GitHub Actions YAML workflow YAML file to target `macos-latest-xlarge` or `macos-13-xlarge`.

Reply:
what happens when they deprecate macos 12...? anyway; yeah that's a mess 😄